src/Hl7.Fhir.Specification/Specification/Snapshot/SnapshotGenerator.cs

```diff
@@ -976,7 +976,10 @@ private async T.Task<bool> mergeTypeProfiles(ElementDefinitionNavigator snap, El
             {
                 // Element specifies multiple type codes, cannot expand children
                 // return false to prevent merging child constraints
-                return false;
+
+                // return false;
+                // [MV 20211123] return true to continue merging diff child constraints
+                return true;
```

Comment:
But` if` and `else if` return true.
But I get that you might want to keep this, in case something else breaks with this commit.

I think Michel's todo` [WMR 20171004] `above explains what should happen.

Reply:
Better would be:
```c#
if (distinctTypeCodeCnt != 1 )
{ 
  return true;
}
else
{
// the rest 
}
```
I will change that